src/strongbox/config.clj

```diff
@@ -107,9 +111,15 @@
   [cfg map?]
   (if (= (->> cfg :catalogue-location-list (map :name) set)
          (->> -default-catalogue-list--v1 (map :name) set))
-    (assoc cfg :catalogue-location-list -default-catalogue-list--v2)
+    (assoc cfg :catalogue-location-list -default-catalogue-list)
     cfg))
 
+(defn-spec remove-curseforge-catalogue map?
+  "removes any mention of the curseforge catalogue from the user config."
+  [cfg map?]
+  (let [new-catalogue-list (vec (remove #(= :curseforge (:name %)) (:catalogue-location-list cfg)))]
```

Comment:
double check we want the keyword `:curseforge` here and not the string used in catalogues